src/sqlfluff/cli/commands.py

```diff
@@ -383,6 +422,35 @@ def dialects(**kwargs) -> None:
     click.echo(format_dialects(dialect_readout), color=c.get("color"))
 
 
+@cli.command()
+@common_options
+@click.option(
+    "--save-path",
+    default=None,
+    type=click.Path(),
+    help="Where to save the completion script. Defaults to ~/.config/sqlfluff",
+)
+@click.argument(
+    "shell_type",
+    default="bash",
+    type=click.Choice(["bash", "zsh", "fish"], case_sensitive=False),
+    # help="What shell to generate autocompletion script for.",
```

Comment:
Did you mean to leave this commented out?

Reply:
yeah you can't have help on a positional argument so will remove that